uvicorn/middleware/proxy_headers.py

```diff
@@ -31,6 +33,12 @@ def __init__(
             self.trusted_hosts = set(trusted_hosts)
         self.always_trust = "*" in self.trusted_hosts
 
+        try:
+            list(map(ipaddress.ip_address, self.trusted_hosts))
+        except ValueError:
+            logger = logging.getLogger("uvicorn.warning")
```

Comment:
We don't have this logger.

```suggestion
            logger = logging.getLogger("uvicorn.error")
```